src/main/java/org/folio/bulkops/domain/converter/ItemNoteListConverter.java

```diff
@@ -36,12 +41,23 @@ public List<ItemNote> convertToObject(String value) {
   public String convertToString(List<ItemNote> object) {
     return  object.stream()
       .filter(Objects::nonNull)
-      .map(itemNote -> String.join(ARRAY_DELIMITER,
-        escape(ItemReferenceHelper.service().getNoteTypeNameById(itemNote.getItemNoteTypeId(), itemNote.getTenantId())),
-        escape(itemNote.getNote()),
-        escape(booleanToStringNullSafe(itemNote.getStaffOnly())),
-        itemNote.getTenantId(),
-        itemNote.getItemNoteTypeId()))
+      .map(itemNote -> {
+        var noteTypeName = itemNote.getItemNoteTypeName();
+        if (isNull(noteTypeName)) {
+          try {
+            noteTypeName = ItemReferenceHelper.service().getNoteTypeNameById(itemNote.getItemNoteTypeId(), itemNote.getTenantId());
+          } catch (NotFoundException e) {
+            log.error("Item note type with id = {} not found : {}", itemNote.getItemNoteTypeId(), e.getMessage());
+            noteTypeName = "";
```

Comment:
Initialise this variable before try-catch block as empty string.

Reply:
Fixed.